.github/workflows/integration-tests-msgraph-tier-0-1.yml

```diff
@@ -0,0 +1,94 @@
+name: Integration tests for MsGraph on Linux - Tier 0 and 1
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_branch:
+        description: 'Base branch'
+        required: true
+        default: 'main'
+  pull_request:
+    paths:
+        - ".github/workflows/integration-tests-msgraph-tier-0-1.yml"
+        - "src/wazuh_modules/wm_ms_graph.h"
+        - "src/wazuh_modules/wm_ms_graph.c"
+        - "src/config/wmodules-ms-graph.c"
+        - "src/Makefile"
+        - "tests/integration/conftest.py"
+        - "tests/integration/test_msgraph/**"
+
+jobs:
+  build:
+    env:
+      BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+      BRANCH_BASE: ${{ github.base_ref || inputs.base_branch }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version-file: ".github/workflows/.python-version-it"
+          architecture: x64
+      # Setup MSGraph Proxy
+      - name: Download m365proxy and config http for msgraph module
+        run: |
+          sed -i 's/https:\/\/%s\/%s/http:\/\/%s\/%s/g' src/wazuh_modules/wm_ms_graph.h
+          curl -L http://packages.wazuh.com/utils/m365proxy/m365proxy.tar.gz | \
+          tar zvx -C /tmp/
+
+      # Build wazuh server for linux.
+      - name: Build wazuh server for linux
+        run: |
+          make deps -C src TARGET=server -j2
```

Comment:
We tried that already, but the agent needs to be connected to a manager to perform the tests correctly, otherwise it stops. So we decided to leave it as manager

Reply:
Use these preload vars:
https://github.com/wazuh/wazuh/blob/master/.github/workflows/integration-tests-github-tier-0-1-lin.yml

Reply:
I used the same as those when I tried the agent

Reply:
Will changue this in another PR.